scripts/new_roman_numerals.js

```diff
@@ -0,0 +1,72 @@
+var pry = require('pryjs')
+function convertToNewRoman(integer){
+  var finalRoman = ""
+
+  var round
+  var remaining
+
+  if (integer < 5) {
+    if (integer == 5-1){
+      finalRoman = "IV"
+    } else {
+    for(i = 1; i <= integer; i++) {
+      finalRoman += "I"
+    };
+  }
+  } else if (integer >= 5 && integer < 10) {
+    round = Math.floor(integer / 5)
+    remaining = integer % 5
+    if (integer == 9){
+      finalRoman = "IX"
+    } else {
+      for(i = 1; i <= round; i++) {
+        finalRoman += "V"
+      }
+      finalRoman += convertToNewRoman(remaining)
+    }
+  } else if (integer >= 10 && integer < 50){
+    round = Math.floor(integer / 10)
+    remaining = integer % 10
+    if (integer == 14) {
+      finalRoman = "XIV"
+    } else if (integer == 19){
+      finalRoman = "XIX"
+    } else {
+      for(i = 1; i <= round; i++) {
+        finalRoman += "X"
+        finalRoman += convertToNewRoman(remaining)
+    }
```

Comment:
that's 100% up to you, but you should line up the ending curly brace with the start of the line the opening one starts on - does that make sense?